pkg/action/executor/limitrange_util.go

```diff
@@ -0,0 +1,317 @@
+package executor
+
```

Comment:
if this is coming from upstream, please drop a proper referene header here. [Example](https://github.com/turbonomic/kubeturbo/blob/master/pkg/discovery/worker/compliance/podaffinity/filtering.go#L2)

Reply:
done